### PR TITLE
feat: add kube-linter tool

### PR DESCRIPTION
**Pull request by tvcsantos**
## Description

This PR adds [kube-linter](https://github.com/stackrox/kube-linter) cli tool to arkade. KubeLinter is a static analysis tool that checks Kubernetes YAML files and Helm charts to ensure the applications represented in them adhere to best practices.

## Motivation and Context

- [ ] I have raised an issue to propose this change, which has been given a label of `design/approved` by a maintainer ([required](https://github.com/alexellis/arkade/blob/master/CONTRIBUTING.md))

I would be nice to have [kube-linter](https://github.com/stackrox/kube-linter) cli tool included on arkade. KubeLinter is a static analysis tool that checks Kubernetes YAML files and Helm charts to ensure the applications represented in them adhere to best practices.

## How Has This Been Tested?

Added tests to `get_test.go` and also run `./hack/test-tool.sh kube-linter`.

If updating or adding a new CLI to `arkade get`, run:

```
go build && ./hack/test-tool.sh TOOL_NAME
```

Logs attached below:

[arkade_kube_linter.log](https://github.com/alexellis/arkade/files/11903975/arkade_kube_linter.log)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Documentation

- [x] I have updated the list of tools in README.md if (required) with `./arkade get --format markdown`
- [ ] I have updated the list of apps in README.md if (required) with `./arkade install --help`

## Checklist:
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/alexellis/arkade/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`

- [ ] I have tested this on arm, or have added code to prevent deployment

### CODE REVIEW

**Comment by alexellis** [comment]
Thanks for the PR. @Jasstkn could you review please?

File: pkg/get/get_test.go [hunk 1]
```diff
@@ -5027,6 +5027,53 @@ func Test_DownloadKubeconform(t *testing.T) {
...
+		},
+		{
+			os:      "linux",
+			arch:    arch64bit,
+			version: version,
+			url:     "https://github.com/stackrox/kube-linter/releases/download/v0.6.4/kube-linter-linux",
+		},
+		{
+			os:      "linux",
+			arch:    archARM64,
```
Comment (Jasstkn) [inline pkg/get/get_test.go:5052]: looks like `kube-linter` doesn't support ARM at all, therefore this test is redundant. 

I think that we need to hardcode the arch64 as the only supported architecture in the template and remove the redundant test. @alexellis what are your thoughts here?

**Comment by Jasstkn** [comment]
@tvcsantos can I also ask you to update your branch from master?

File: pkg/get/get_test.go (hunk 1 above)
Comment (tvcsantos) [reply pkg/get/get_test.go:5052]: You are right I will remove it ;)

**Comment by tvcsantos** [comment]
All done. If you need something extra ping :)

**Review by Jasstkn — APPROVED** [review]
LGTM 🚀

**Comment by alexellis** [comment]
@tvcsantos just need a rebase on the README file now, thanks.

**Comment by tvcsantos** [comment]
> @tvcsantos just need a rebase on the README file now, thanks.

Done ;)

**Comment by tvcsantos** [comment]
> > @tvcsantos just need a rebase on the README file now, thanks.
> 
> Done ;)

okay not done. I will do it in a bit :D. Having some troubles with the signoff :D

**Comment by tvcsantos** [comment]
Now all good :). If you need something ping me 👍

**Comment by alexellis** [comment]
Looks like we have 6x commits now, so if you follow my video it will walk you through how to squash them all down.

But before you do that, rebase master with `git pull origin master -r` - or if master is your branch, add an upstream remote:

```
git remote add upstream https://github.com/alexellis/arkade
git pull upstream master -r
```

**Comment by tvcsantos** [comment]
> Looks like we have 6x commits now, so if you follow my video it will walk you through how to squash them all down:
> 
> https://www.youtube.com/watch?v=8j0H6urZ-bU
> 
> But before you do that, rebase master with `git pull origin master -r` - or if master is your branch, add an upstream remote:
> 
> ```
> git remote add upstream https://github.com/alexellis/arkade
> git pull upstream master -r
> ```

Okay I got it you only want one commit. Done ;) sorry for the delay

**Comment by alexellis** [comment]
Thanks Tiago. Looking forward to your next PR.

**Pull request merged**